api/v1beta1/ibmpowervsmachine_types.go

```diff
@@ -74,10 +74,12 @@ type IBMPowerVSMachineSpec struct {
 // a validation error.
 type IBMPowerVSResourceReference struct {
 	// ID of resource
+	// +kubebuilder:validation:MinLength=1
```

Comment:
can we set this `+kubebuilder:validation:MinLength=1` for the `ServiceInstanceID` as well?